### ISSUE TITLE
Remove commented code

### ISSUE THREAD

**Issue by chelofm** [MetalGeorge/UnitTestingProjectUASB#2]
https://github.com/MetalGeorge/UnitTestingProjectUASB/blob/8dd81e0cd92d31789f4a18a453940d2904b95ce5/src/test/java/TestReadFile.java#L17

As you know it is not good practice have commented code

**Comment by MetalGeorge** [comment]
true, issue was when executing in travis, there the file is not being recognized
